File: introducao-a-computacao/TrabalhoProgramacao/operacoes-aritmeticas.cpp
#include <iostream>
using namespace std;
	
int main() 
{
  int opcao=0, resposta;
  float num1, num2, resultado;

  for (int i=0;i<999;i++) 
  {
    cout << "Olá!\nDigite a operação desejada: 1 Soma  2 Subtração  3 Multiplicação  4 Divisão\n";
    cin >> opcao;
    
    switch (opcao) 
    {
      case 1: 
        cout << "Digite o primeiro número: ";
        cin >> num1;
        
        cout << "Digite o segundo número: ";
        cin >> num2;
        
        resultado=num1+num2;
        
        cout << "O resultado é " << resultado << "."; 
        break;
      
      case 2: 
        cout << "Digite o primeiro número: ";
        cin >> num1;
        
        cout << "Digite o segundo número: ";
        cin >> num2;
        
        resultado=num1-num2; 
        
        cout << "O resultado é " << resultado << "."; 
        break;
      
      case 3: 
        cout << "Digite o primeiro número: ";
        cin >> num1;
        
        cout << "Digite o segundo número: ";
        cin >> num2;
        
        resultado=num1*num2;
        
        cout << "O resultado é " << resultado << "."; 
        break;
      
      case 4: 
        cout << "Digite o primeiro número: ";
        cin >> num1;
        
        cout << "Digite o segundo número: ";
        cin >> num2;
        
        resultado=num1/num2;
        
        cout << "O resultado é " << resultado << "."; 
        break;
      
      default: 
        cout << "Erro. Tente Novamente.";
        break;
    }
    
    cout << "Deseja repetir? 1 Sim 2 Não\n";
    cin >> resposta;

    if (resposta==1) 
    {
      i++;
    }
    else 
    {
      i=999;
      cout << "Obrigada! Até a próxima!";
    }
	
	}
	return 0;
}

File: introducao-a-computacao/TrabalhoProgramacao/operacoes-aritmeticas.py
opcao = int(input('Olá!\nDigite a operação desejada: 1 Soma  2 Subtração  3 Multiplicação  4 Divisão '))

while i<999:
 if opcao==1:
    num1 = float(input('Digite o primeiro número: '))
    num2 = float(input('Digite o segundo número: '))
    resultado=num1+num2
    print("O resultado é", resultado, ".")

  elif opcao==2:
    num1 = float(input('Digite o primeiro número: '))
    num2 = float(input('Digite o segundo número: '))
    resultado=num1-num2
    print("O resultado é", resultado, ".")

  elif opcao==3:
    num1 = float(input('Digite o primeiro número: '))
    num2 = float(input('Digite o segundo número: '))
    resultado=num1*num2
    print("O resultado é", resultado, ".")

  elif opcao==4:
    num1 = float(input('Digite o primeiro número: '))
    num2 = float(input('Digite o segundo número: '))
    resultado=num1/num2
    print("O resultado é", resultado, ".")

  else:
    print("Erro. Tente Novamente.")
 
resposta = int(input('Deseja repetir? 1 Sim 2 Não'))
  if resposta==1:

  else:
    i=999
    print('Obrigada! Até a próxima!')